Octokit/Models/Response/PullRequest.cs

```diff
@@ -242,6 +243,11 @@ public bool Merged
         /// Users requested for review
         /// </summary>
         public IReadOnlyList<User> RequestedReviewers { get; protected set; }
+        
+        /// <summary>
+        /// Teams requested for review
+        /// </summary>
+        public IReadOnlyList<Team> RequestedTeams { get; protected set; }
```

Comment:
Given we have `RequestedReviewers` above that is configured in the same way, I think this is all we need to do.

Is there any other feedback you're looking for @ch1seL?

Reply:
This is half the success of issue #2122
I think we can merge it but the issue will not be resolved totally

Reply:
Now I understand. I'll merge this and add some more context to #2122 about the second item on the list once I know what needs to change.